docs/UserGuide.md

```diff
@@ -4,339 +4,1044 @@
+After hitting `Enter`, you have successfully included supplementary information which you need to remember about the 
+internship, making it a valuable resource when preparing for interviews or follow-ups.
 
 ### Step 5: Start Using LetsGetHired
 
-**Congratulations**! You've successfully managed your internship application using LetsGetHired. These steps demonstrate the core functions of the application, helping you stay organized and informed throughout your internship application journey.
+**Congratulations**! You have successfully managed your first internship application using **LetsGetHired**. 
+These steps demonstrate the core functions of the application, helping you stay organized and informed throughout your internship application journey.
+
+<box type="info" seamless>
 
---------------------------------------------------------------------------------------------------------------------
+For other commands, please refer to the [_Commands_ section of the **User Guide**](#commands) below.
+
+</box>
+
+__________________________________________________________________________________________________________________
+
+<div style="page-break-after: always;"></div>
 
 ## **Commands**
 
 This section explains the detailed list of the commands (and its usages) which are available for you to use.
 
-<box type="info" seamless>
+<box type="tip" seamless>
 
-**Notes about the command format:**<br>
+If you have been using **LetsGetHired** and just need a **quick refresher** on the commands available, you can jump to 
+the [_Command Summary_ section of the **User Guide**](#command-summary) below.
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/COMPANY_NAME r/ROLE c/CYCLE`, `COMPANY_NAME`, `ROLE` and `CYCLE` are parameters which can be used as `add n/Google r/Full Stack Developer c/Summer 2024`.
+</box>
 
-* Items in square brackets are optional.<br>
-  e.g. `c/CYCLE [s/STATUS]` can be used as `c/Summer s/Accepted` or as `c/Summer`.
+<box type="info" light>
 
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `c/CYCLE r/ROLE`, `r/ROLE c/CYCLE` is also acceptable.
+**Notes about the command format:**<br>
 
-* Irrelevant parameters for commands that do not take in parameters (such as `help` and `list`) will be ignored.<br>
+* Words in `UPPER_CASE` are **parameters** which you will type in.<br>
+    ```
+    add n/COMPANY_NAME r/ROLE c/CYCLE
+    ``` 
+    For example, you can replace `COMPANY_NAME`, `ROLE` and `CYCLE` as follows:
+    ```
+    add n/Google r/Software Engineering c/Summer 2024
+    ```
+
+* All parameters should be preceded by its respective **prefix**.<br>
+    ```
+    add n/COMPANY_NAME r/ROLE c/CYCLE
+    ``` 
+    For example, `n/` is the prefix for the `COMPANY_NAME` parameter. `r/` is the prefix for `ROLE` and `c/` is the prefix for `CYCLE`.
+
+    <box type="warning" light>
+  <md>
+  **Exception**: The `INDEX` parameter does not require any prefix.
+  </md>
```

Comment:
Can include an empty line after </md>

![Screenshot 2023-11-01 at 3 15 34 PM](https://github.com/AY2324S1-CS2103T-W17-2/tp/assets/80398765/9cd40f27-9288-48fd-9e0b-6095190155de)

Reply:
I think this is an issue on IntelliJ's preview! I ran it on MarkBind and based all my formatting from there.
![image](https://github.com/AY2324S1-CS2103T-W17-2/tp/assets/107602488/dcf02d57-2234-4c96-895c-0c5d73dbc873)
Here is a screenshot!